### DISCUSSION TITLE
Where can I get the css file for ruby plug-in?

### DISCUSSION THREAD

**Discussion by KITAMURA-Tomihiro** [Alex-D/Trumbowyg#1421] [Q&A]
I wanted to use the ruby plug-in for Trumbowyg, but I could not find css file for it that is explaied in "Demos | Trumbowyg: A lightweight WYSIWYG editor | Alex-D / Alexandre Demode」<https://alex-d.github.io/Trumbowyg/demos/#plugins-ruby>. 

Could someone tell me where to get the css file?

**Comment by Alex-D** [comment]
That's an error in the documentation since there is no custom CSS for the Ruby plugin.
